README.md

```diff
@@ -231,6 +231,7 @@ In addition to the bindings above, third party developers have started to provid
 Install the required dependencies using `pip3`:
 
 ```bash
+sudo apt-get install build-essential libssl-dev libffi-dev python-dev
```

Comment:
I'd suggest rather that we remove paramiko dep, it's not really required anymore.

Reply:
Just clarifying, that means removing `bin/benchmark_nc.py` as well, right?

Reply:
@reuben We could keep it and move the paramiko dependency out of the default scope